src/ActionExecutor/UserConfirmation.php

```diff
@@ -184,7 +235,16 @@ protected function doFinal(View $modal)
         $this->_jsSequencer($modal, $this->jsGetExecute($return, $this->action->owner->id));
     }
 
-    protected function jsGetExecute($obj, $id)
+    /**
+     * Return proper js statement when action execute.
+     *
+     * @param $obj
+     * @param $id
+     *
+     * @return array
+     * @throws \atk4\core\Exception
+     */
+    protected function jsGetExecute($obj, $id): array
```

Comment:
At least in many other atk files, protected/private functions are prefixed with an underscore, like `_jsGetExecute`. Is that still the way to go? Personally I like this naming convention.

Reply:
@PhilippGrashoff  - My understanding is that we are moving toward PSR-12 standard which forbid using a single underscore char in front of a method name.

> Method names MUST NOT be prefixed with a single underscore to indicate protected or private visibility. That is, an underscore prefix explicitly has no meaning.

Reply:
Ah, I was not aware of this rule in PSR12. Thanks for the explanation!